wave_controls/src/pid.cpp

```diff
@@ -0,0 +1,51 @@
+#include "wave/controls/pid.hpp"
+
+namespace wave {
+
+PID::PID(void) {
+    this->error_prev = 0.0;
```

Comment:
Fewer lines of code (especially when you have multiple, slightly different, constructors as here), less likely to forget to initialize something.